integration-tests/pipelines/e2e-tests-kind-pipeline.yaml

```diff
@@ -0,0 +1,326 @@
+  params:
+    - name: SNAPSHOT
+      description: 'The JSON string representing the snapshot of the application under test.'
+      default: '{"components": [{"name":"test-app", "containerImage": "quay.io/example/repo:latest"}]}'
+      type: string
+    - name: test-name
+      description: 'The name of the test corresponding to a defined Konflux integration test.'
+      default: ''
+      type: string
+    - name: ocp-version
+      description: 'The OpenShift version to use for the ephemeral cluster deployment.'
+      type: string
+    - name: test-event-type
+      description: 'Indicates if the test is triggered by a Pull Request or Push event.'
+      default: 'none'
+    - name: konflux-test-infra-secret
+      description: The name of the secret where testing infrastructure credentials are stored.
+      type: string
+    - name: replicas
+      description: 'The number of replicas for the cluster nodes.'
+      type: string
+    - name: machine-type
+      description: 'The type of machine to use for the cluster nodes.'
+      type: string
+    - name: oci-container-repo
+      default: 'quay.io/konflux-test-storage/konflux-team/release-service-catalog'
+      description: The ORAS container used to store all test artifacts.
+    - name: quality-dashboard-api
+      default: 'none'
+      description: 'Contains the url of the backend to send metrics for quality purposes.'
+    - name: component-image
+      default: 'none'
+      description: 'Container image built from any konflux git repo. Use this param only when you run Konflux e2e tests
+        in another Konflux component repo. Will pass the component built image from the snapshot.'
+    - name: pipeline-test-suite
+      type: string
+      description: 'The name of the test corresponding to a defined Konflux integration test.'
+    - name: pipeline-used
+      default: ''
+      type: string
+      description: |
+        The pipeline that is used by the test suite. 
+        If empty, use the pipeline-test-suite as the pipeline to be used
+  tasks:
+    - name: test-metadata
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/tekton-integration-catalog.git
+          - name: revision
+            value: main
+          - name: pathInRepo
+            value: common/tasks/test-metadata/0.1/test-metadata.yaml
+      params:
+        - name: SNAPSHOT
+          value: $(params.SNAPSHOT)
+        - name: test-name
+          value: $(context.pipelineRun.name)
+    - name: check-if-related-test
+      when:
+        - input: "$(tasks.test-metadata.results.pull-request-author)"
+          operator: notin
+          values: ["red-hat-konflux[bot]"]
+      runAfter:
+        - test-metadata
+      taskRef:
+        resolver: git
+        params:
+          - name: url
+            value: https://github.com/konflux-ci/release-service-catalog
+          - name: revision
+            value: main
```

Comment:
we don't use the main branch in this repo (but I don't think this task should live here anyways)

Reply:
Changed.